dev-libs/mini-xml/mini-xml-2.12.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit autotools
+
+MY_PN="mxml"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="A small XML parsing library that you can use to read XML data files or strings"
+HOMEPAGE="https://github.com/michaelrsweet/mxml
+	https://www.msweet.org/mxml/"
+SRC_URI="https://github.com/michaelrsweet/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+LICENSE="Mini-XML"
```

Comment:
The license in the git repo has a newer date. Could you diff them?

Reply:
Yes. Thanks for the hint. There are two differences.

1) It's now under LGPL2 instead of LGPL.
2) Linking with it must use LGPL2 also.

In this case, I will update the license directly, since it already was part of 2.10 (oldest version in portage).